src/Analysis/AnalysisApp.jsx

```diff
@@ -52,6 +55,22 @@ class AnalysisApp extends React.Component {
     queryClient.invalidateQueries('workflows');
   }
 
+  processAppMessages = (event) => {
+    const pathArray = this.state.app.applicationUrl.split( '/' );
+    const protocol = pathArray[0];
+    const host = pathArray[2];
+    const applicationBaseUrl = protocol + '//' + host;
+
+    // ONLY process messages coming from the same domain as the app AND
+    // which contain the message "refresh token!":
+    if (event.origin === applicationBaseUrl &&
+        event.data === "refresh token!") {
+        console.log( "received: " + event.data );
```

Comment:
this is ok since it is merging into a dev branch, let's remove it bbefore merging into master

Reply:
I'll remove it now, np.

Reply:
done.